File: src/codes/Users_Manual/modulation/csv_0.cpp
//~#include<autd3.hpp>
#include "autd3/modulation/audio_file.hpp"

//~int main() {
//~using namespace autd3;
const auto path = "path/to/foo.csv";
modulation::audio_file::Csv(path, 4000.0f * Hz,
                            modulation::audio_file::CsvOption{
                                .delimiter = ',',
                            });
//~return 0; }

File: src/codes/Users_Manual/modulation/csv_0.rs
# use autd3::prelude::*;
use autd3_modulation_audio_file::{Csv, CsvOption};

# fn main() {
# let _ = 
Csv::new(
    "path/to/foo.csv",
    4000.0 * Hz,
    CsvOption { delimiter: b',', has_headers: false },
);
# }